data-providers/data-provider-base/src/main/java/datart/data/provider/ProviderManager.java

```diff
@@ -214,7 +215,8 @@ private void excludeColumns(Dataframe data, Set<SelectColumn> include) {
                     .noneMatch(selectColumn ->
                             column.columnKey().equals(selectColumn.getColumnKey())
                                     || column.columnKey().equals(selectColumn.getAlias())
-                                    || column.columnKey().contains(selectColumn.getColumnKey()))) {
+                                    // 用正则做聚合函数权限的判断 剔除原来的contains判断
```

Comment:
hi @vip2193 should we avoid to using Chinese? cc @scottsut

Reply:
ok i will change to use english